cmd/update-main.go

```diff
@@ -98,26 +97,39 @@ func GetCurrentReleaseTime() (releaseTime time.Time, err error) {
 	return getCurrentReleaseTime(Version, os.Args[0])
 }
 
-func isDocker(cgroupFile string) (bool, error) {
-	cgroup, err := ioutil.ReadFile(cgroupFile)
-	if os.IsNotExist(err) {
-		err = nil
+// Check if we are indeed inside docker.
+// https://github.com/moby/moby/blob/master/daemon/initlayer/setup_unix.go#L25
+//
+//     "/.dockerenv":      "file",
+//
+func isDocker(dockerEnvFile string) (ok bool, err error) {
+	_, err = os.Stat(dockerEnvFile)
+	if err != nil {
+		if os.IsNotExist(err) {
+			err = nil
+		}
+		return false, err
 	}
-
-	return bytes.Contains(cgroup, []byte("docker")), err
+	return true, nil
```

Comment:
My bad. I was trying to say returning bool and have `errorIf()` here.  Its ok to continue this way to avoid error message printing when unit test is run.